### PR TITLE
Adjust rich-text source mentions in docs

### PR DESCRIPTION
**Pull request by ellatrix**
## Description

After #10439, adjusts the docs to suggest 'html' source instead of 'rich-text'.

## Checklist:
- [ ] My code is tested.
- [ ] My code follows the WordPress code style. 
- [ ] My code follows the accessibility standards. 
- [ ] My code has proper inline documentation.

### CODE REVIEW

File: docs/block-api/attributes.md [hunk 1]
```diff
@@ -55,26 +55,17 @@ Use `html` to extract the inner HTML from markup.
...
 	}
 }
-// {
-//   "content": [
-//     "Vestibulum eu ",
-//     { "type": "strong", "children": "tortor" },
-//     " vel urna."
-//   ]
-// }
+// { "content": "<p>Quoted text</p><p>And more</p>" }
```
Comment (tofumatt) [inline docs/block-api/attributes.md]: Maybe using an example like `"<p>Quoted text</p><p>Second line</p>"` might make this a bit more immediately obvious.

**Review by tofumatt — APPROVED** [review]
Makes sense to me! I suggested a tweak but feel free to make it if it makes sense, then merge.

**Comment by gziolo** [comment]
It's tricky to assign proper milestone :D As it is synced with Gutenberg handbook up to 15 minutes after the merge.

**Comment by iseulde** [comment]
@gziolo Not sure I understand... could you elaborate?

**Comment by gziolo** [comment]
Please check https://wordpress.org/gutenberg/handbook/block-api/#attributes-optional - your changes are already there. The flow for documentation is not ideal. That's why we still put milestone according to the plugin release schedule. Next release is 4.1 as 4.0-RC can get only patches before it gets out. Ideally, Gutenberg handbook should show docs for 3.9 release, 4.0-RC as an option and `development` (your changes) as yet another option.

**Comment by gziolo** [comment]
BTW, @pento @youknowriad - do we have anything planned to fix the issue with the content of Gutenberg handbook as 5.0 goes out?

**Comment by iseulde** [comment]
Wow. Before it's even merged?

**Comment by gziolo** [comment]
> Before it's even merged?

I merged PR, if this is what you ask :)

**Pull request merged**